### PR TITLE
AAP-7349 Add e2e test for websocket

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**
Add E2E test for websocket
Depends on https://github.com/ansible/ansible-rulebook/pull/234
Closes https://issues.redhat.com/browse/AAP-7349

### CODE REVIEW

**Comment by Alex-Izquierdo** [comment]
Also added @hsong-rh Because I know that he worked on this functionality

File: tests/e2e/files/rulebooks/websockets/test_websocket_range.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- name: Test websocket range events
+  hosts: all
+  sources:
+    - name: Generate a range
+      benthomasson.eda.range:
```
Comment (bzwei) [inline tests/e2e/files/rulebooks/websockets/test_websocket_range.yml:5]: should it be `ansible.eda.range`?

Comment (Alex-Izquierdo) [reply tests/e2e/files/rulebooks/websockets/test_websocket_range.yml]: Good catch. Fixed.

**Comment by benthomasson** [comment]
@Alex-Izquierdo please rebase

**Review by benthomasson — APPROVED** [review]

**Review by hsong-rh — APPROVED** [review]

**Review by ddonahue007 — APPROVED** [review]
lgtm

**Pull request merged**